packages/devextreme/js/__internal/ui/stepper/stepper.ts

```diff
@@ -44,16 +46,11 @@ export const ORIENTATION: Record<string, Orientation> = {
   vertical: 'vertical',
 };
 
-export interface StepperProperties extends Properties {
-  selectionMode?: SingleOrNone;
-
-  loopItemFocus?: boolean;
-
-  selectionRequired?: boolean;
-
+export interface StepperProperties extends Properties, Omit<
```

Comment:
Perhaps more correctly change it as follows 

```suggestion
export interface StepperProperties extends CollectionWidgetEditProperties<Stepper, Item>, Omit<Properties<Item>, keyof CollectionWidgetEditProperties<Stepper, Item>>
```
Kindly think about it